test/support/factories.rb

```diff
@@ -229,6 +229,9 @@
     end
   end
 
+  factory :devolved_administration_availability do
+  end
```

Comment:
```suggestion
  factory :devolved_administration_availability
```

I doubt you need the block here.

Reply:
👍